layers/+lang/c-c++/funcs.el

```diff
@@ -225,28 +225,27 @@ and the arguments for flyckeck-clang based on a project-specific text file."
   (spacemacs//c-c++-lsp-set-config (intern (concat "c-c++-lsp-" suffix)) nil (concat "-" suffix)))
 ;; -- END helper functions for common configuration of cquery and ccls backends
 
-
-(defun spacemacs//c-c++-lsp-enable ()
-  "Enable the LSP backend specified by the `c-c++-backend' configuration variable."
-    (progn (condition-case nil
-             (spacemacs//c-c++-lsp-call-function "lsp-" "-enable")
-             (user-error nil))))
-
 (defun spacemacs//c-c++-lsp-config ()
   "Configure the LSP backend specified by the `c-c++-backend' configuration variable."
     (progn
-      (spacemacs//c-c++-lsp-setup-company)
       (spacemacs//c-c++-lsp-define-extensions)
       (spacemacs//c-c++-lsp-wrap-functions)
       (setq-default flycheck-disabled-checkers '(c/c++-clang c/c++-gcc))
 
+      (dolist (param '("executable" "extra-init-params" "project-whitelist" "project-blacklist" "sem-highlight-method"))
```

Comment:
I think two additional variables should be mapped from lsp to ccls/cquery here. Firstly, `initialization-options` to mirror `extra-init-params` which ccls doesn't use anymore. Secondly, `extra-args` to allow, for instance, adding logging from spacemacs configuration instead of creating a wrapper script that would add additional arguments. If you would add them though, they need to be defined in config.el

```suggestion
      (dolist (param '("executable" "extra-args" "extra-init-params" "initialization-options" "project-whitelist" "project-blacklist" "sem-highlight-method"))
```

Reply:
Also done